list-to-add.txt

```diff
@@ -1,4 +1,9 @@
-Milad Mirg
-Valkyrae
-Sykunno
-Digusised Toast (if he still makes videos)
+Michael Reeves
+Dude Perfect
```

Comment:
```suggestion
```
Can't add this as its a company name now a individual's

Reply:
K on it.